packages/block-library/src/legacy-widget/block.json

```diff
@@ -5,6 +5,9 @@
 		"widgetClass": {
 			"type": "string"
 		},
+		"id": {
```

Comment:
I just removed this attribute in another PR, but without it there's no way to send it over through `ServerSideRender`. In short - we need to **have** this attribute, but we shouldn't **use** it :-) A much better solution would be making the server side rendering API less restrictive - custom arguments would be incredibly helpful (and a great follow-up to this PR).

Reply:
Can we rename it "widgetId" or something like that. "id" is too generic and can potentially be used for something else that works across blocks like HTML ids. or block ids...